appveyor.yml

```diff
@@ -9,7 +9,7 @@ environment:
         - php_ver: 7.0.9
 
 cache:
-    - '%APPDATA%\Composer'
+    - '%LOCALAPPDATA%\Composer\files'
```

Comment:
I just saw it on another project where Composer cache seems to work, but just `'%LOCALAPPDATA%\Composer'` would be fine to me too. I'll remove it once the current build will be finished so we know whether this really improves something.